src/applications/terms-and-conditions/tests/containers/MhvTermsAndConditions.unit.spec.jsx

```diff
@@ -30,14 +32,20 @@ describe('<MhvTermsAndConditions>', () => {
   };
 
   const setup = () => {
-    global.window.location.replace = sinon.spy();
+    oldLocation = global.window.location;
```

Comment:
changes to JSDOM prevent us from directly overriding the `location` object's properties so we need to delete it and replace it for our testing purposes.

Reply:
Do you think we could set `window.location` to `writeable:true` similar to what you are doing below and what's mentioned in https://github.com/facebook/jest/issues/890?